lib/Lifters/BasicBlockLifter.cpp

```diff
@@ -481,12 +481,23 @@ BasicBlockFunction BasicBlockLifter::CreateBasicBlockFunction() {
     return this->ProvidePointerFromFunctionArgs(func, index);
   };
 
-
   LOG(INFO) << "Live values at entry to function "
             << this->block_context->LiveBBParamsAtEntry().size();
   this->UnpackLiveValues(ir, ptr_provider, this->state_ptr,
                          this->block_context->LiveBBParamsAtEntry());
 
+  for (auto &reg_const : this->block_context->GetConstants()) {
+    auto new_value = this->options.program_counter_init_procedure(
```

Comment:
If it's not too much effort we probably want to make this conditioned on a flag here: https://github.com/trailofbits/irene3/blob/4381e2cf8d318628d88a9c80a0fe02310b09268f/data_specifications/specification.proto#L160 ie. the spec should specify wether to construct a plain integer constant or not. If this isn't too much effort for you would be good to have, otherwise we can merge this and Ill address it in my upcoming rework of equalities

Reply:
Ive decided that we'll address this in the tail call representation PR